### ISSUE TITLE
Since 3.21.0 icons in grid layout have less than the usual vertical padding on some pages

### ISSUE THREAD

**Issue by CanisLupus** [CanisLupus/swift-selection-search#97]
Originally reported at issue #92 (thanks, @mazesy!).

In the 3.21.0 update, in order to support variable-width separators without distorting the icon, each icon _img_ is now contained in its own html element. This seems to cause some issues with sites with older DOCTYPE declarations.

From [this list](https://www.w3schools.com/tags/tag_doctype.asp), the following cause the problem:

**HTML 4.01 Transitional**
`<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">`

**HTML 4.01 Frameset**
`<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Frameset//EN" "http://www.w3.org/TR/html4/frameset.dtd">`

**XHTML 1.0 Transitional**
`<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">`

**XHTML 1.0 Frameset**
`<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Frameset//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-frameset.dtd">`

**Comment by CanisLupus** [comment]
Fixed in v3.21.1, unless someone finds otherwise :p

The solution was awesome, of course. x) Just position all icons manually on the popup and don't let the natural HTML+CSS positioning happen.

The advantage was that I could remove the wrappers elements from the icons again since I now position everything exactly as needed. Without the wrappers and page-specific positioning rules, the DOCTYPE problem doesn't happen anymore.

Leaving this open for a while until it is confirmed as solved. :)

**Comment by mazesy** [comment]
Nice work! @CanisLupus 
Just installed SSS 3.21.1 and the vertical padding issue appears resolved after having tested on the problematic sites.

There's one very slight difference between left and right popup paddings, and that appears everywhere, not specific to a site. Here's a demo pic:

![firefox_screenshot_2018-05-31t21-51-34 256z](https://user-images.githubusercontent.com/31442711/40810457-1dd272f2-652e-11e8-8f0b-24dc5f56db6b.png)

In my case, given my icon padding, I have 11px on the left and 3px on the right...
But it's really a detail!

**Comment by CanisLupus** [comment]
Wow, I have no idea how I missed that, but things like this _have_ happened before without me noticing, so.... yeah. ;)

Thanks a lot, @mazesy! You've been very helpful reporting these bugs. :)

I submitted v3.21.2 to fix that and also made the separators respect the padding setting. They didn't before, but I changed my mind (it shouldn't affect you since you don't use them in the popup).

Cheers!
Daniel
